index.html

```diff
@@ -29,14 +29,14 @@
             </tr>
             <!--            second person -->
             <tr>
-                <td>Firstname Lastname</td>
-                <td>Artist Name</td>
-                <td>State Name</td>
-                <td>Number of Siblings</td>
-                <td><a class="btn btn__female">Female</a></td>
+                <td>Ngutor Mase</td>
```

Comment:
Hi @MaseDavid well done, you did the right thing however, the branch you are working on is an old version and thus is conflicting with the new one.

Firstly, you need to update your version of the repo ours by running git fetch. Afterwards, you can effect your changes and any conflict. You are on the right track. Almost there :)